lib/cc/services/github_pull_requests.rb

```diff
@@ -13,13 +13,35 @@ class Config < CC::Service::Config
       label: "Github Context",
       description: "The integration name next to the pull request status",
       default: "codeclimate"
+    attribute :welcome_comment_enabled, Axiom::Types::Boolean,
+      label: "Welcome comment enabled?",
+      description: "Should Code Climate post a welcome comment on pull requests?",
```

Comment:
Might want to get some copy team opinions here, but personally based on what I see in other integrations, I think we could go with something briefer. Maybe `"Post a welcome comment?"`.